src/templates/zone.ejs

```diff
@@ -1,5 +1,9 @@
 <div class="overlay" data-zone="<%= zone.id %>" style="display:none">
     <div class="cards">
+        <div class="audio-player">
+            <span class="close-deck">&times;</span>
```

Comment:
I'm told that all the decks will have audio so I don't think this will be a real issue. I'd be more inclined to change the name of the div id to be more generic.